src/components/Hexagons/HexagonGrid.svelte

```diff
@@ -0,0 +1,44 @@
+<script lang="ts">
+  // Inspired by https://github.com/sveltejs/svelte-virtual-list/blob/master/VirtualList.svelte
+  type T = $$Generic;
+
+  export let items: T[];
+  export let cols: number;
+
+  export let gap: 'small' | 'medium' | 'big' = 'medium';
+</script>
+
+<!-- Warning: Very magic, totally eyeballed numbers in the next line -->
```

Comment:
warning makes sense, maybe a little less goofy xd

Reply:
Okay, I've changed the comment. Let me know what you think!